ci/checkpoint.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Command line arguments
+if [ "$ne" == '1' ]
+then
+  echo "Usage: $0 <verilator, vcs>"
+  exit 1
+elif [ $1 == "vcs" ]
+then
+    SUFFIX=v
+elif [ $1 == "verilator" ]
+then
+    SUFFIX=sc
+else
+  echo "Usage: $0 <verilator, vcs>"
+  exit 1
+fi
+
+# Default to 1 core
+N=${2:-1}
+
+# Bash array to iterate over for configurations
+cfgs=(\
+    "e_bp_unicore_cfg"
+    "e_bp_multicore_1_cfg"
+    "e_bp_multicore_1_cce_ucode_cfg"
+    )
+
+let JOBS=${#cfgs[@]}
+let CORES_PER_JOB=${N}/${JOBS}+1
+
+# The base command to append the configuration to
+cmd_base="make -j ${CORES_PER_JOB} -C bp_top/syn build.v sim_sample.v SUITE=beebs PROG=aha-compress CHECKPOINT_P=1 SAMPLE_START_P=1000 SAMPLE_MEMSIZE=64"
```

Comment:
This won't work for multithreading since it'll trying sim-ing before build.  I would remove CORES_PER_JOB as a variable